src/Microsoft.AspNetCore.Blazor.Build/Microsoft.AspNetCore.Blazor.Build.csproj

```diff
@@ -19,6 +19,14 @@
     <None Remove="tools\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- This is a newer version than is in the shared framework, so copy it -->
+    <Content Include="..\Microsoft.AspNetCore.Blazor.Razor.Extensions\bin\$(Configuration)\netstandard2.0\Microsoft.AspNetCore.Razor.Language.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </Content>
+  </ItemGroup>
```

Comment:
This is the kind of thing that pops up when you're using the build output to create a CLI tool. Any package reference you have that diverges from the shared framework won't be around for you to use. 

Technically the output of dotnet build is not designed to be portable, that's supposed to require a publish. In the long run for Blazor this shouldn't matter if we can just use the Razor build server.

Reply:
Glad you're anticipating that sort of thing!